### DISCUSSION TITLE
EventCountLimit cannot be set to more than 128 in Opentelemetry auto instrumentation using java

### DISCUSSION THREAD

**Discussion by mounikaVeeha** [open-telemetry/opentelemetry-java-instrumentation#8055] [Q&A]
Hi , We have a requirement where for each trace id we need to set 1000 events for one trace .But as per the documentation we see that we can only set events size to 128 . Is there a way where we could increase size based on the requirement we have.

**Comment by laurit** [comment]
Have you tried using the system property or environment variable described in https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#span-limits to increase the limit?
